### DISCUSSION TITLE
Get-PnPSearchCrawlLog cmd and data interpretation

### DISCUSSION THREAD

**Discussion by GitGrr123** [pnp/powershell#2254] [Q&A]
Hi all, 

Could anybody explain me how to interpret data received from Get-PnPSearchCrawlLog cmd, please? Especially, what information is provided by:
-  CrawlTime
-  ItemTime
-  SPItemModifiedTime
-  TimeStamp

**Comment by kmarwen** [comment]
Hi there 
i'm looking for the same thing
maybe when seeing the code : 
https://github.com/pnp/PnP-PowerShell/blob/master/Commands/Search/GetSearchCrawlLog.cs#L218

ItemTime is **ItemDate**
CrawlTime is : CrawlTime = (DateTime)dictionary["**TimeStampUtc**"]
